browser/src/simulations/noop_scenario.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {handlePreferenceCheckbox, performLogin} from './login.js';
+import {postInChannel} from './post_in_channel.js';
+import {scrollInChannel} from './scrolling_in_channel.js';
+import type {BrowserInstance} from '../lib/browser_manager.js';
+
+// NoopScenario is a very basic scenario.
+// It just performs very basic actions in a loop. Such as:
+// Login, Post in channel, Scroll in channel and then repeat the process.
+export async function noopScenario({page, userId, password}: BrowserInstance, serverURL: string, runInLoop = true) {
```

Comment:
Nit: three operations don't look like a "noop" to me 😂 What about `postAndScrollScenario` or something similar?

Reply:
Yes makes sense